src/Sarif.UnitTests/MockFactory.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Moq;
+
+using Match = System.Text.RegularExpressions.Match; // Avoid ambiguity with Moq.Match;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal static class MockFactory
+    {
+        public static IFileSystem MakeMockFileSystem(string fileName, string fileText = null, string[] fileLines = null)
```

Comment:
Any reason not to just pass `fileText`? Your mock `Setup` for `ReadAllLines` could split it, and you're reduce your storage requirements by half.

Reply:
ok

---
In reply to: [202750290](https://github.com/Microsoft/sarif-sdk/pull/945#discussion_r202750290) [](ancestors = 202750290)